gfx/MShockXotto+/pngs_tiles_32x32/character/overlay/mutations/hair/overlay_female_mutation_hair.json

```diff
@@ -1,620 +0,0 @@
-[
-  {
-    "id": "overlay_female_mutation_hair_crewcut_var_black",
-    "fg": [ "overlay_female_mutation_hair_black_crewcut" ],
```

Comment:
this part should not have been removed, as you can see the id and fg field there are not the same.